src/llscan.cc

```diff
@@ -61,7 +61,7 @@ bool FindObjectsCmd::DoExecute(SBDebugger d, char** cmd,
   std::sort(sorted_by_count.begin(), sorted_by_count.end(),
             TypeRecord::CompareInstanceCounts);
 
-  uint64_t total_objects = 0;
+  uint64_t total_objects = 0, total_size = 0;
```

Comment:
Minor nit: do you mind splitting this into two separate declarations?

Reply:
No problem. PR updated with this change

Reply:
Thank you!